### ISSUE TITLE
Insiders feature: onGlobalParticipantChanged

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#489]
A single listener for all participant changed events in all groups.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.67](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.66...1.9.67) (2020-06-10)

- build(deps-dev): bump @types/node from 14.0.11 to 14.0.12 [`#532`](https://github.com/open-wa/wa-automate-nodejs/pull/532)
- Release 1.9.67 [`7f1ddbb`](https://github.com/open-wa/wa-automate-nodejs/commit/7f1ddbb25421bca453db7bde897431b92da3d9d2)
- updated docs [`ffcf94c`](https://github.com/open-wa/wa-automate-nodejs/commit/ffcf94c2edf6e464fe2bc6cba702e81a6a059852)
- feature: Implemented `onGlobalParicipantsChanged`#489 [`0724491`](https://github.com/open-wa/wa-automate-nodejs/commit/0724491067506572c70c450d522fe26e71fdda87)

**Comment by smashah** [comment]
With `onGlobalParticipantsChanged`, you no longer have to add a specific group id.

https://open-wa.github.io/wa-automate-nodejs/classes/client.html#onglobalparicipantschanged

```javascript
client.onGlobalParicipantsChanged(change=>console.log(change))
```
